web-console/knikubevirt/expose-secret/expose-secret.md

```diff
@@ -0,0 +1,48 @@
+#Expose Secret, Config Maps and Service Accounts as Disks
+
+Kubernetes recently introduced three new volume types (Secrets, Config Maps and Service Accounts). These volumes can be used as additional VM disks, and allow better integration of Virtual Machines with Kubernetes resources.
+We will try and define where is the best place for the user to handle these disks.
+
+A similar process already exists when attaching Secrets or Config Maps to a deployment config. This happens in the “Environment” section under deployment and deployment config. The Secret/Config Map can be added as an environment variable or as a volume.
+The idea is to be consistent by doing similar processes in the same way across the tool. To support that, we added an “Environment” tab under the VM to handle the attached Secrets, Config Maps and Service Accounts
+
+## Add “Environment” tab to handle the attached Secrets, Config Maps and Services Account
+
+![added environment tab](img/environment-tab.png)
+
+The user can add more Secrets/Config Maps/Service Accounts:
+
+![add](img/add.png)
+
+![add dropdown open](img/add-dropdown-open.png)
+
+## Expose From Secrets/Config Maps/Service Accounts Lists
+
+![expose from accounts list](img/expose-from-accounts-list.png)
+
+![add to workload](img/add-to-workload.png)
+
+We added a new action to the kebab Menu called “Add to Workload”.
+
+![add to workload popup](img/add-to-workload-popup.png)
+
+![add to workload popup open dropdown](img/add-to-workload-popup-open-dropdown.png)
```

Comment:
Later on we say that the "Disk" option will only be available is a VM is chosen as the workload. We should probably update this to either show a VM being selected first or remove Disk as an option at this point.

Reply:
@lizsurette, I am not sure I understand this because we show that on the add-secret-to-vm.png

Reply:
I guess the question is... Should VM be shown in the dropdown at this point? There hasn't been a workload selected yet.

Reply:
Thanks, got it now :)